roles/cobbler_profile/tasks/import_distro.yml

```diff
@@ -0,0 +1,88 @@
+---
+# This profile will do all the work necessary to create a new distro/profile
+# pair in Cobbler.
+
+# Since this profile will be used several times in the same playbook,
+# mention the distro name each time.
+- name: Distro name
+  debug: var=distro_name
```

Comment:
I guess that's fine, as along as it'll give us a good message why it failed.

I just didn't want this to silently noop or something because of human error.